contracts/src/l1/TokenBridge.sol

```diff
@@ -168,15 +181,30 @@ contract TokenBridge is
     function deposit(address l1Token_, uint256 l2Recipient, uint256 amount) onlyApprovedToken(l1Token_) external {
         IStaticATokenLM l1Token = IStaticATokenLM(l1Token_);
         l1Token.transferFrom(msg.sender, address(this), amount);
-        sendMessage(l1Token_, msg.sender, l2Recipient, amount);
+
+        (bool success, bytes memory result) = l1Token_.call(abi.encodeWithSignature("_getCurrentRewardsIndex()"));
```

Comment:
why use this low-level call when we have the interface of the l1Token_ ?

Reply:
_getCurrentRewardsIndex is not in the interface yet